addons/l10n_ar_pos/views/templates.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+
+    <template id="ticket_validation_screen" inherit_id="point_of_sale.ticket_validation_screen">
+
+        <!-- We do not let the user create the invoice by then self. That is why we hide the "Get my Invoice" button -->
+        <xpath expr="//div[@id='get_my_invoice']" position="attributes">
+            <attribute name="t-if">pos_order.company_id.country_code != 'AR'</attribute>
+        </xpath>
+        <xpath expr="//div[@id='get_my_invoice']/.." position="after">
+            <h4 class="text-danger" t-if="pos_order.company_id.country_code == 'AR' and not pos_order.is_invoiced">Invoice not available. You can contact us for more info</h4>
+        </xpath>
+
+        <!-- If the user is NOT log in we should hide the partner form, this one lost functionality because the get my invoice button is not shown and then the partner info is never save. In they want they can sign up and fill data from there -->
+        <xpath expr="//div[hasclass('o_portal_details')]" position="attributes">
+            <attribute name="t-if">pos_order.company_id.country_code != 'AR'</attribute>
+        </xpath>
+        <xpath expr="//div[@id='get_info_div']" position="attributes">
+            <attribute name="t-if">pos_order.company_id.country_code != 'AR'</attribute>
+        </xpath>
+        <xpath expr="//div[@id='get_info_div']" position="after">
+                <t t-else="">
+                    <h4> Please
+                    <a role="button" t-att-href="'/web/login?redirect=/pos/ticket/validate?access_token=%s' % access_token" style="margin-top: -11px"> Sign in</a>
+                    in order to save your contact info</h4>
+                </t>
+        </xpath>
+
+        <!-- Show AR fields preview info, and show warning is a required field need to be completed -->
+        <xpath expr="//t[@id='partner_vat']" position="before">
+            <t t-if="pos_order.company_id.country_code == 'AR'">
+                <t t-if="partner.l10n_ar_afip_responsibility_type_id">
+                    <t t-out="partner.l10n_ar_afip_responsibility_type_id.name"/><br />
+                </t>
+                <t t-else="">
+                    <span class="text-danger">* Please configue your AFIP Responsability</span><br />
+                </t>
+                <t t-if="partner.l10n_latam_identification_type_id">
+                    <t t-out="partner.l10n_latam_identification_type_id.name" /> -
+                </t>
+                <t t-else="">
+                    <span class="text-danger">* Please configue your Identifcation Type</span><br />
+                </t>
+            </t>
+        </xpath>
+        <xpath expr="//t[@id='partner_vat']" position="after">
+            <t t-else="">
+                <span class="text-danger">* Please configue your Identifcation Number</span><br />
+            </t>
+        </xpath>
```

Comment:
We tested it but we cannot access this. Is it accessible and if yes can you provide us the steps ?

Reply:
Hello @adgu-odoo this was actually related to the PR where the customer could edit their info in the portal view, but that was removed from this PR and we will work on that later. As of now, this is not needed.

Hello @jjscarafia could you help us remove this part as it has nothing to do with this PR? Let me know when it's done so I can test and I'll give you feedback (if any)... 

Cheers!

Fernando

@vbe-odoo @ren-odoo @zaoral @caburj @pablohmontenegro

Reply:
removed!
@fvz-odoo

Reply:
Thanks @jjscarafia 

I just tested this and it is still working correctly. @adgu-odoo this is ready.

Cheers team!

@vbe-odoo @zaoral @ren-odoo @caburj